smartmontools.mk

```diff
@@ -0,0 +1,44 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
```

Comment:
Can you add an `ifneq` to build for macOS targets only, since it does not work on iOS currently? Reference: [dimentio.mk](https://github.com/ProcursusTeam/Procursus/blob/as2/dimentio.mk).